packages/react-dev-utils/InlineChunkHtmlPlugin.js

```diff
@@ -29,8 +29,8 @@ class InlineChunkHtmlPlugin {
   }
 
   apply(compiler) {
-    let publicPath = compiler.options.output.publicPath;
-    if (!publicPath.endsWith('/')) {
+    let publicPath = compiler.options.output.publicPath || '';
+    if (publicPath.length > 0 && !publicPath.endsWith('/')) {
```

Comment:
This doesn't seem right. We always want the trailing slash in this scenario, right?
```suggestion
    if (!publicPath.endsWith('/')) {
```

Reply:
OK, but let's make it consistent with the other truthy check:
This doesn't seem right. We always want the trailing slash in this scenario, right?
```suggestion
    if (publicPath && !publicPath.endsWith('/')) {
```

Reply:
Ok I merged it